addons/mail/static/src/core/web/sidebar.xml

```diff
@@ -54,7 +54,7 @@
         <div class="flex-grow-1"/>
         <div class="d-flex me-3">
             <i t-if="category.canView" t-attf-class="fa fa-cog {{ hover_class }}" title="View or join channels" t-on-click="() => this.openCategory(category)" role="img"/>
-            <i t-if="category.canAdd and category.isOpen" class="o-mail-DiscussCategory-add" t-attf-class="fa fa-plus {{ hover_class }} ms-1" t-on-click="() => this.addToCategory(category)" t-att-title="category.addTitle" role="img"/>
+            <i t-if="category.canAdd and category.isOpen" class="o-mail-DiscussCategory-add" t-attf-class="fa fa-plus {{ hover_class }} ms-1" t-on-click="() => this.addToCategory(category)" t-att-title="category.addTitle" role="img" t-att-data-hotkey="category.addHotkey"/>
```

Comment:
So when category is folded, control panel does not suggest to add item in this category?
It's alright, but can feel weird for end-users that sometimes it's visible and sometimes not, as fold/unfolding a category is more for cleaning UI and it's not expected to influence shortcut to action.

Reply:
I think this is technical stuff, the hotkey can only bind to a button existing on the page. It feels off for a user to use hotkey to open the page and another hotkey to add a channel/chat...

Reply:
It feels a bit off that the action to add items requires the category being unfolded.
We did that so when it's folded there are less buttons on it, and we can click anywhere on title to open it.
Let's keep it as it is, it's not a big deal :)